asset/src/drop_field/processor.ts

```diff
@@ -0,0 +1,31 @@
+import {
+    MapProcessor,
+    OpConfig,
+    DataEntity,
+} from '@terascope/job-components';
+import DataWindow from '../__lib/data-window';
+
+export default class DropField extends MapProcessor<OpConfig> {
+    map(doc: DataEntity): DataEntity {
+        if (doc instanceof DataWindow) {
+            return this.handleDataWindow(doc);
+        }
+
+        this.dropField(doc);
+
+        return doc;
+    }
+
+    private handleDataWindow(doc: DataWindow): DataWindow {
+        doc.dataArray = doc.asArray().map((item: DataEntity) => {
+            this.dropField(item);
+            return item;
+        });
+
+        return doc;
+    }
+
+    private dropField(doc: DataEntity) {
+        delete doc[this.opConfig.field];
+    }
+}
```

Comment:
This `drop_field` is effectively a replacement for `drop_property` no?  Does that imply that we're deprecating the other one, if so, perhaps we should think about how to do that.  Should we warn the users when they submit jobs using `drop_property`?  Then eventually remove `drop_property` altogether?

Reply:
Right, that's why I couldn't find `drop_property` being deleted.